### ISSUE TITLE
NTR: Synovial biopsy: synovial hyperplasia (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9350]
New term request Synovial biopsy: synovial hyperplasia (athrey_140130130752): Synovial lining cells increased

**Comment by pnrobinson** [comment]
@MickeySegal
Is the difference between plasia and trophy clinically important or can we add this as a broad synonym to 

[Synovial hypertrophy HP:0005186](https://hpo.jax.org/app/browse/term/HP:0005186)

**Comment by MickeySegal** [comment]
Seem close enough to me.

**Comment by pnrobinson** [comment]
@MickeySegal 
I am changing the primary label of this term to 
Synovial lining hyperplasia
def:  Synovial hyperplasia involves proliferation of mesenchymal stromal/stem cells (MSCs) and leads to synovial thickening, which can be observed radiographically.
PMID:27412524
PMID:30618151


Comment: Synovial hyperplasia is a frequent clinical finding in patients with joint injury and other joint disorders.
